src/pages/home/page.jsx

```diff
@@ -2,16 +2,10 @@ import React from "react";
 import { getData } from "../../common/request";
 
 
-let HomePage = React.createClass({
-  statics: {
-    fetchData(params) {
-      return getData("/home");
-    }
-  },
-
+export default class HomePage extends React.Component {
   componentWillMount() {
     console.log("[HomePage] will mount with server response: ", this.props.data.home);
```

Comment:
> Is this equivalent?

I think it's very similar, but [bind](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind) may handle variable arguments slightly differently and takes any arbitrary object to bind to, not just `this`:

```js
var func = func.bind({random: "Object"});
```